hack/errata.py

```diff
@@ -90,6 +92,58 @@ def notify(message, webhook=None):
     }).encode('utf-8'))
 
 
+def get_open_prs_to_fast(repo):
+    pulls = repo.get_pulls(state='open', sort='created', base='master')
+    for pr in pulls:
```

Comment:
Nit: you don't save much by defining `pulls`.  Maybe `for pull in repo.get_pulls(...):`?